test/test_formatters.rb

```diff
@@ -19,7 +19,7 @@ def setup(formatter_class)
     @empty_twine_file = Twine::TwineFile.new
     @formatter = formatter_class.new
     @formatter.twine_file = @empty_twine_file
-    @formatter.options = { consume_all: true, consume_comments: true }
+    @formatter.options = { consume_all: true, consume_comments: true, developer_language: 'en' }
```

Comment:
Instead of doing this globally in setup, can you just do this in your new tests below?

Reply:
indeed ! thanks